app/views/transactions/_form.html.slim

```diff
@@ -13,8 +13,9 @@
           collection: current_organization.categories.grouped_by_type,
           group_method: :last, prompt: 'Category', label: false, value_method: :id
     .col-md-3.col-sm-6.col-xs-6
-      = f.input :customer_id, as: :select, include_blank: 'Customer',
-        label: false, collection: current_organization.customers
+      = f.input :customer_name, label: false,
+        input_html: { data: { url: autocomplete_customers_path,
+          placeholder: f.object.customer_name || 'Customer'} }
```

Comment:
@yunixon you should not use customer name as placeholder. You should set it as predefined value. There is an option in select2 for it.